rsrcdump/pict.py

```diff
@@ -1,31 +1,37 @@
+from typing import Dict, List, Tuple, Union
+
 from ctypes import ArgumentError
 import io
-from rsrcdump.packutils import Unpacker
 from dataclasses import dataclass
 import struct
 
+from rsrcdump.packutils import Unpacker
+
+
 class PICTError(Exception):
     pass
 
 class Xmap:
+    __slots__ = ('frame_r', 'frame_l', 'frame_b', 'frame_t',
+                 'rowbytes', 'pixelsize')
     @property
-    def frame_w(self):
-        return self.frame_r - self.frame_l
+    def frame_w(self) -> int:
+        return self.frame_r - self.frame_l# type: ignore
```

Comment:
"type: ignore" won't be necessary if we use the field notation above.

```suggestion
        return self.frame_r - self.frame_l
```